### PR TITLE
Removed redundant and unnecessary calls to save().

### PR DESCRIPTION
**Pull request by DeathByTape**
PR in response to: https://github.com/yahoo/elide/issues/170

### CODE REVIEW

**Comment by wcekan** [comment]
:+1:

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by wcekan** [comment]
@DeathByTape Found a problem here.  Will need to revert.

File: elide-core/src/main/java/com/yahoo/elide/Elide.java [hunk 1]
```diff
@@ -126,6 +126,7 @@ public ElideResponse get(
             ElideResponse response = buildResponse(responder.get());
             auditLogger.commit();
             transaction.commit();
+            requestScope.saveObjects();
```
Comment (wcekan) [inline elide-core/src/main/java/com/yahoo/elide/Elide.java]: @DeathByTape Missed this.  saveObjects occurs **after** commit.

**Pull request merged**